make/test_target_script.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/sh
+
+# 
+# %CopyrightBegin%
+# 
+# Copyright Ericsson AB 1997-2019. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# 
+# %CopyrightEnd%
+#
+
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+LIGHT_CYAN='\033[1;36m'
+BOLD='\033[1m'
+NC='\033[0m'
+
+
+print_highlighted_msg_with_printer () {
+    COLOR=$1
+    MSG_PRINTER=$2
+    printf "\n${COLOR}======================================================================${NC}\n"
+    echo
+    $MSG_PRINTER
+    echo
+    printf "${COLOR}======================================================================${NC}\n"
+}
+
+print_highlighted_msg () {
+    COLOR=$1
+    MSG=$2
+    print_msg () {
+        echo "$MSG"
+    }
+    print_highlighted_msg_with_printer $COLOR print_msg
+}
+
+print_all_tests_takes_long_time_warning () {
+    print_msg () {
+        cat << EOM
+
+WARNING
+
+All tests will require several hours to run. You may want to check the
+following text file that describes how to run tests for a specific
+application.
+
+EOM
+        echo $ERL_TOP/HOWTO/TESTING.md
+    }
+    print_highlighted_msg_with_printer $YELLOW print_msg
+}
+
+print_all_tests_for_application_notes () {
+    print_msg () {
+        cat << EOM
```

Comment:
This text is very verbose.... maybe only display it the first time "make test" is used in an application?

Reply:
The message is only displayed on first invocation with the commit below.